automata/tm/tape.py

```diff
@@ -58,6 +67,14 @@ def move(self, direction):
             current_position=new_position
         )
 
+    def clone(self):
```

Comment:
It did behave a little bit different, since the `copy` method in the `Automaton` class would not create a proper copy of a `MNTM`, because my `__init__` method didn't include a copy of the `tapes` list, but only an assignment. Now that I have changed this in my `__init__` method by setting `self.tapes = [tape.copy() for tape in tapes]`, every test works ok. Without this change, they don't.

Reply:
@CamiloMartinezM so are you saying if you name the method `copy`, that things will break? Because it's not a problem for me if you override the parent class's method here, but I would prefer if the function name were kept as `copy()` instead of `clone()`. That was all.

Reply:
Yes, it did break, but I had not thought of making the copy of `tapes` adequately. Now that I did, I can use the `copy` method from the `Automaton` class and I can delete `clone()`.